docs/migration-guides/qiskit-2.0.mdx

```diff
@@ -0,0 +1,260 @@
+
+5. The input combination where a custom `coupling_map` is provided with either a `backend` or `basis_gates`
+  that contain gates with 3 or more qubits is no longer allowed. The coupling map
+  does not provide the necessary connectivity details to be able to determine the  action of the gate.
+  In these cases, the recommended migration path is the creation of a custom target from scratch.
+
+6. The `inst_map` argument has been removed without alternative. It was used to provide pulse information
+  to the transpiler, and the full `qiskit.pulse` module has been removed.
+
+
+<span id="qiskit.pulse"></span>
+## qiskit.pulse
+
+The module `qiskit.pulse` has been completely removed from Qiskit, without replacement.
+This includes the related items:
+
+- the [`QuantumCircuit`](/api/qiskit/qiskit.circuit.QuantumCircuit), [`DAGCircuit`](/api/qiskit/qiskit.dagcircuit.DAGCircuit) and [`DAGDependency`](/ap/qiskit/qiskit.dagcircuit.DAGDependency) property `calibrations`.
+- the [`QuantumCircuit`](/api/qiskit/qiskit.circuit.QuantumCircuit) methods `has_calibration_for` and `add_calibration`.
+- the [`DAGCircuit`](/api/qiskit/qiskit.dagcircuit.DAGCircuit) method `has_calibration_for`.
+- the transpiler passes `PulseGates`, `RXCalibrationBuilder`, `RZXCalibrationBuilder`, `EchoRZXWeylDecomposition`, `ValidatePulseGates`.
+- the module `qiskit.scheduler` and all its contents.
+- the [`BackendV2`](/api/qiskit/qiskit.providers.BackendV2) methods `instruction_schedule_map`, `drive_channel`, `measure_channel`, `acquire_channel` and `control_channel`.
+- the [`Target`](/api/qiskit/qiskit.transpiler.Target) methods `has_calibration`, `get_calibration`, `instruction_schedule_map` and `update_from_instruction_schedule_map`.
+- the `inst_map` argument of [`generate_preset_pass_manager`](/api/qiskit/qiskit.transpiler.generate_preset_pass_manager), [`transpile`](/api/qiskit/qiskit.compiler.transpile), and [`Target.from_configuration`](/api/qiskit/qiskit.transpiler.Target#qiskit.transpiler.Target.from_configuration).
+- the `calibration` field of [`InstructionProperties`](/api/qiskit/qiskit.transpiler.InstructionProperties).
+
+This happened because IBM Quantum devices ceased to offer pulse-level access, and the Qiskit team no longer had resources to maintain the little used package.
+You can continue to use the `qiskit.pulse` module while using the 1.x series of Qiskit, which has bugfix support until September 2024 and security support until March 2025, but the module is removed from the 2.x series.
+
+Most uses of `qiskit.pulse` can be replaced with direct access to fractional gates on IBM Quantum Heron QPUs (and later generations, if applicable).
+You can read more in [the migration guide for converted pulse usage to fractional gates](/migration-guides/pulse-migration).
+
+You can no longer load QPY files that contain `ScheduleBlock` objects in Qiskit 2.0 because of the removal of Qiskit Pulse.
+You can continue to load these files in the Qiskit 1.x series.
```

Comment:
```suggestion
You can continue to use the `qiskit.pulse` module while using the v1.x series of Qiskit, which has bugfix support until September 2024 and security support until March 2025, but the module is removed from the v2.x series.

Most uses of `qiskit.pulse` can be replaced with direct access to fractional gates on IBM Quantum Heron QPUs (and later generations, if applicable).
You can read more in [the migration guide for converted pulse usage to fractional gates](/migration-guides/pulse-migration).

You can no longer load QPY files that contain `ScheduleBlock` objects in Qiskit v2.0 because of the removal of Qiskit Pulse.
You can continue to load these files in the Qiskit v1.x series.
```